### ISSUE TITLE
Potentially corrupt transactions blob for ledger #45173109

### ISSUE THREAD

**Issue by Mwni** [XRPLF/clio#1249]
When trying to retrieve the transactions of ledger #45173109 from a clio node, it throws `Internal error.` (code 73).
The snippet below can be executed in a browser console to confirm:
```js
(() => {
    const socket = new WebSocket('wss://s2.ripple.com')
    socket.onmessage = event => console.log(event.data) && socket.close()
    socket.onopen = () => socket.send(JSON.stringify({ 
        command: 'ledger',
        ledger_index: 45173109,
        transactions: true,
        expand: true
    }))
})()
```
Attached is the output from rippled for this ledger.
[ledger-45173109.json](https://github.com/XRPLF/clio/files/14548143/ledger-45173109.json)

**Comment by cindyyan317** [comment]
@Mwni Thanks for reporting this 🌷 . We have fixed the corruption for 45173109. Please try again. We are working on a DB validation tool to avoid such issue happening again. 

Let us know if problem still exists.

**Comment by Mwni** [comment]
@cindyyan317 Yes, that did it. Thanks alot!
